tools/rapids-prompt-local-repo-config

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# A utility script that prompts user to setup repository information in
+# local environments
+
+if [ "${CI:-false}" = "false" ]; then
+  rapids-echo-stderr "Local run detected."
+  rapids-echo-stderr "NVIDIA VPN connectivity is required to download workflow artifacts."
+
+  if [ -z "${RAPIDS_BUILD_TYPE:-}" ]; then
+    {
+      echo ""
+      read -r -p "Enter workflow type (one of: pull-request|branch|nightly): " RAPIDS_BUILD_TYPE
+      export RAPIDS_BUILD_TYPE
+      echo ""
+      echo "Suppress this prompt in the future by setting the 'RAPIDS_BUILD_TYPE' environment variable:"
+      echo "export RAPIDS_BUILD_TYPE=${RAPIDS_BUILD_TYPE}"
+      echo ""
+    } >&2
+  fi
+
+  if [ -z "${RAPIDS_REPOSITORY:-}" ]; then
+    {
+      echo ""
+      read -r -p "Enter org/repository name (e.g. rapidsai/cudf): " RAPIDS_REPOSITORY
+      export RAPIDS_REPOSITORY
+      echo ""
+      echo "Suppress this prompt in the future by setting the 'RAPIDS_REPOSITORY' environment variable:"
+      echo "export RAPIDS_REPOSITORY=${RAPIDS_REPOSITORY}"
+      echo ""
+    } >&2
+  fi
+
+  if [ -z "${RAPIDS_REF_NAME:-}" ]; then
+    {
+      echo ""
+      if [ "${RAPIDS_BUILD_TYPE}" = "pull-request" ]; then
```

Comment:
```suggestion
      if [ "${RAPIDS_BUILD_TYPE:-pull-request}" = "pull-request" ]; then
```

If we remove the `RAPIDS_BUILD_TYPE` default value as mentioned below, we should probably re-add it here to preserve the existing functionality.

- https://github.com/rapidsai/gha-tools/pull/57/files/f33c72eb82ef7d8f5bbfe61e0c794fc3d28025d3#r1268473429

Reply:
I agree, I would prefer to remove it altogether.

Reply:
I'm fine with this :+1: